mind-meld.js

```diff
@@ -4,28 +4,37 @@ Meteor.methods({
     check(collectionName, String);
     check(password, String);
 
-    if (!Meteor.settings.MIND_MELD_TOKEN)
-      throw new Meteor.Error('no token set');
+    if (!Meteor.settings.mindmeld || !Meteor.settings.mindmeld.password)
+      throw new Meteor.Error('no password set in Meteor.settings.mindmeld.password');
 
-    if (password !== Meteor.settings.MIND_MELD_TOKEN)
-      throw new Meteor.Error('incorrect password: ' + password);
+    if (password !== Meteor.settings.mindmeld.password)
+      throw new Meteor.Error('incorrect export password');
 
     return MindMeld.export(collectionName);
   },
 
+  mm_import(options) {
```

Comment:
Wrap the body of this Meteor Method in a publicly available method called `MindMeld.import`, to keep users from using strings and `Meteor.call` which is an annoyance.

`MindMeld.import` should delegate to `MindMeld._import` after checking pwds and such.

Some for `MindMeld.export`